hydra-gradle-plugin/src/main/resources/META-INF/gradle-plugins/com.pandora.hydra.properties

```diff
@@ -1 +1 @@
-implementation-class=com.pandora.hydra.HydraPlugin
\ No newline at end of file
+implementation-class=HydraPlugin
```

Comment:
It should be fully qualified name, plus we need to make sure these meta files are unique for two plugins